GOVERNANCE.md

```diff
@@ -0,0 +1,68 @@
+# CRI-O contributor tiers
```

Comment:
Let's link the file from the README.md with a short sentence describing its intention.

Reply:
Good thought @saschagrunert I'd also suggest a link and a bit of explanatory verbiage on CONTRIBUTING.md